src/api/index.js

```diff
@@ -15,11 +15,25 @@ export const checkEmail = (email) => logined.get(`user/email-check?email=${email
 
 export const changeNickname = (nickName) => logined.post('user/nickname', nickName);
 
+export const sendFindPassword = ({ account, email }) => logined.post('user/email-send/PASSWORD', { account, email });

```

Comment:
대관님 해당 함수의 역할에 비해 **네이밍**이 이 조건을 충족하지 못하는 것 같습니다.
_아래와 같이 수정하는 것은 어떨까요?_

```suggestion
export const 대관이바보 = ({ account, email }) => logined.post('user/email-send/PASSWORD', { account, email });
```